app/src/main/java/org/mozilla/tv/firefox/utils/ServiceLocator.kt

```diff
@@ -84,7 +85,9 @@ open class ServiceLocator(val app: Application) {
     val screenshotStoreWrapper by lazy { PinnedTileImageUtilWrapper(app) }
     val formattedDomainWrapper by lazy { FormattedDomainWrapper(app) }
     val channelRepo by lazy { ChannelRepo(pinnedTileRepo) }
+    val pocketEndpointRaw by lazy { PocketEndpointRaw(appVersion, buildConfigDerivables.globalPocketVideoEndpoint) }
```

Comment:
This doesn't seem like something that should be publicly exposed.  Looking at the finished code, `PocketEndpointRaw` is only used by `PocketEndpoint` and `PocketVideoFetchWorker`.  If we update `PocketVideoFetchScheduler` and `PocketVideoFetchWorker` to accept a `PocketEndpointRaw` as a param, `pocketEndpointRaw` could be a private param in `ServiceLocator`.

Reply:
Agreed.

Reply:
Passing parameters to a `Worker` is more complicated because it needs to be passed as a `Data` object. Typically, only basic types like strings and ints are passed as data, not objects. Not sure if this is worth the complexity? Is there another way?

Reply:
I don't understand, what do you mean about it needing to be passed as a data object?  

We could also talk about this on Slack or Zoom.